src/decoration.h

```diff
@@ -10,6 +10,7 @@
 class Client;
 class Settings;
 class DecorationScheme;
+class XConnection;
```

Comment:
Why is this needed in this change?

Reply:
Thanks! That was a mistake (I was already at the next step...). I've removed the line!

Reply:
Thanks!